decentraland/unity-explorer
# feat: profiles

### What?

Introduces the fetch of profile information in json format and then applying your own avatar through the debug panel.

### How to test

1. Run the `DynamicSceneLoading` scene
2. Select a realm, i recommend [peer.decentrand.org](http://peer.decentrand.org/) since it has the real profiles
3. In the debug menu you will see an item: PROFILE: AVATAR SHAPE
4. Copy any wallet address
5. Click Set Address button (twice, seems like there is something wrong in the debug panel)
6. You should see your avatar changing after a short delay
